application/api/capacityservice/tests/test_serializers/test_payload_serializer.py

```diff
@@ -33,8 +63,25 @@ def test_reset_time(self):
             "Returned reset data time is not as expected.",
         )
 
+    def test_reset_time_3(self):
+        "Test reset_time method for GREEN status update"
+
+        payload_data = {
+            "capacityStatus": "GREEN",
+            "resetStatusIn": 300,
+        }
+        current_time = datetime.now()
+
+        returned_reset_time = CapacityStatusRequestPayloadSerializer._resetTime(
+            self, current_time, payload_data
+        )
+
+        self.assertIsNone(
+            returned_reset_time, "Returned reset data time is not None.",
+        )
+
     def test_convert_to_model_1(self):
```

Comment:
As there is only one of these test I think the number in the name isn't necessary

Reply:
Agree